app/views/admin/legislation/processes/index.html.erb

```diff
@@ -20,6 +20,8 @@
         <th class="text-center"><%= t("admin.legislation.processes.process.start_date") %></th>
         <th class="text-center"><%= t("admin.legislation.processes.process.end_date") %></th>
         <th class="text-center"><%= t("admin.legislation.processes.process.comments") %></th>
+        <th class="text-center"><%= SDG::Goal.model_name.human(count: 2).upcase_first %></th>
```

Comment:
This is (I think) a mistake I've made as well :sweat_smile:. Instead of `count: 2`, I think `count: :other` is better since there are languages which have a different plural rules depending on the number of elements :thinking:.